lib/y2_module_basetest.pm

```diff
@@ -31,8 +31,25 @@ our @EXPORT = qw(is_network_manager_default
   continue_info_network_manager_default
   accept_warning_network_manager_default
   workaround_suppress_lvm_warnings
+  with_yast_env_variables
 );
 
+=head2 with_yast_env_variables
+
+ with_yast_env_variables([extra_vars]);
+
+Set environment variables for yast application.
+C<extra_vars> extends the variables that we can use. C<extra_vars> expects a string.
```

Comment:
Hi Yannis! Could you please change phrasing to be impersonal.

For example, instead of:
> C<extra_vars> extends the variables that we can use.

use:
C<extra_vars> extends the variables that can be used.

Reply:
Other than that, LGTM.

Reply:
Done